android/app/src/main/kotlin/coop/polypoly/polypod/polyNav/PolyNav.kt

```diff
@@ -77,7 +76,7 @@ open class PolyNav(
                 }
                 val newId = UUID.randomUUID().toString()
                 val fs = Preferences.getFileSystem(context).toMutableMap()
-                fs[fsPrefix + newId] = fileName
+                fs[PolyOut.fsPrefix + newId] = fileName
```

Comment:
This is again the "polypod://id..." thing. I wonder if we should also centralize that, and have a way of generating and validating it, as well as encapsulating it into an object. Maybe make it part of the `Meta` interface?

Reply:
Or maybe a new polyPath or polyUri?

Reply:
Also, we might want to coordinate this with the triple namespaces used in polyIn. So maybe it should be part of polyIn after all.